src/parser/scanner.lex

```diff
@@ -60,6 +71,15 @@ LABEL_FULL_WIDTH            {CN_EN_FULL_WIDTH}{CN_EN_NUM_FULL_WIDTH}*
 
 %%
 
+ /* Flex rules section */
```

Comment:
ditto.

Reply:
Flex doesn't allow `//` used and, and `/*` must be indented.